ImprovedAnvil/lib/spells/03_patches_vanilla.tph

```diff
@@ -1297,6 +1297,7 @@ COPY_EXISTING ~writhing.pro~ ~override/writhing.pro~
 
   BUT_ONLY
 
+// @todo
```

Comment:
needs a description as well, to remember what needs to be done :) @TODO is just for ease of search later

Reply:
this long list of strings looks like it doesn't need any comments to see what needs to be done :) but I provided a short description